probePlugin/src/main/scala/org/virtuslab/ideprobe/handlers/RunConfigurations.scala

```diff
@@ -182,6 +149,65 @@ object RunConfigurations extends IntelliJApi {
     new RunnerSettingsWithProcessOutput(settings)
   }
 
+  private def availableRunConfigurations(
+                                          scope: TestScope
+                                        )(implicit ec: ExecutionContext): java.util.List[ConfigurationFromContext] = {
+    BackgroundTasks.withAwaitNone {
```

Comment:
Formatter behaves strangely in such cases, I would format it by hand for now.

Reply:
Fixed